Pear/Controller/LoginViewController.swift

```diff
@@ -155,8 +128,9 @@ extension LoginViewController: GIDSignInDelegate {
             return
         }
 
-        if let email = user.profile.email,
-           !(email.contains("@cornell.edu")) && email != "cornellpearapp@gmail.com" {
+        guard let email = user.profile.email,
+              // Only allow users with Cornell emails or internal users (Pear email) to log in
+              email.contains("@cornell.edu") || email == "cornellpearapp@gmail.com" else {
```

Comment:
and just for the record it's okay not to use Apple login for our app even with the new guidelines
https://developer.apple.com/forums/thread/124006#:~:text=Sign%20in%20with%20Apple%20is,existing%20education%20or%20enterprise%20account.

Reply:
ooo great!